src/content/getting-started/_index.en.md

```diff
@@ -70,11 +70,8 @@ tested with the following network (CNI) Plugins:
 
 ## Deployment
 
-The available methods for deployment are:
+Submariner is deployed and managed using its Operator. [Submariner's Operator](https://github.com/submariner-io/submariner-operator) can be
+deployed using [subctl](../operations/deployment), [Helm](../operations/deployment/helm), or directly.
 
-* [subctl](../operations/deployment)
-* [Operator](https://github.com/submariner-io/submariner-operator)
-* [Helm](../operations/deployment/helm)
-
-`subctl` greatly simplifies the deployment of Submariner, and is therefore the recommended deployment
-method.
+`subctl` is the recommended deployment method because it has the most refined deployment user experience and additionally provides testing,
+debugging, and bug-reporting capabilities.
```

Comment:
"bug-reporting capabilities" makes it sound like you can report a bug directly from the tool?

Reply:
“bug-diagnosing” might be more accurate.

Reply:
I meant that you can gather everything required to report a bug, I can try to make that more clear.

Reply:
Yes, I understand that, just trying to make it clearer for someone who is not familiar with the tool.

Reply:
> “bug-diagnosing” might be more accurate.

This wasn't loading in for me when I originally commented, but yes that's great wording IMHO. Updated accordingly.